src/v2/providers/alerts/crashlytics.ts

```diff
@@ -0,0 +1,325 @@
+import { defineEndpoint, FirebaseAlertData } from '.';
+import { ManifestEndpoint } from '../../../common/manifest';
+import { CloudEvent, CloudFunction } from '../../core';
+import * as options from '../../options';
+
+/** Data */
+/** Generic issue interface */
+interface Issue {
+  id: string;
+  title: string;
+  subtitle: string;
+  appVersion: string;
+}
+/** Payload for a new fatal issue */
+export interface NewFatalIssuePayload {
+  ['@type']: 'com.google.firebase.firebasealerts.CrashlyticsNewFatalIssuePayload';
+  issue: Issue;
+}
+/** Payload for a new non-fatal issue */
+export interface NewNonfatalIssuePayload {
+  ['@type']: 'com.google.firebase.firebasealerts.CrashlyticsNewNonfatalIssuePayload';
+  issue: Issue;
+}
+/** Payload for a regression alert */
+export interface RegressionAlertPayload {
+  ['@type']: 'com.google.firebase.firebasealerts.CrashlyticsRegressionAlertPayload';
+  type: string;
+  issue: Issue;
+  resolveTime: string;
```

Comment:
FYI: I don't recall all the details, but we have an outstanding bug for the auth namespace that the proto-JSON format isn't the SDK format for date strings. Let's not make the same mistake again.

Reply:
https://github.com/firebase/firebase-functions/issues/1012 for details. Let's bring this up with API council and see if there needs to be an AIP on the standard way going forwards.